tracing-subscriber/src/lib.rs

```diff
@@ -74,6 +74,8 @@
 //!   than `Vec`) as a performance optimization. Enabled by default.
 //! - [`parking_lot`]: Use the `parking_lot` crate's `RwLock` implementation
 //!   rather than the Rust standard library's implementation.
+//! - `wasm-bindgen`: Uses the [`web-time`] crate to allow `Uptime` and `SystemTime` to
+//!   be used in browsers.
```

Comment:
On the other hand, I suppose if we want to add additional WASM support, such as using `Console.log` with more structured values when running in the browser, we may want to have one big feature flag for that.

Reply:
> it seems a bit strange to me that the feature flag is `wasm-bindgen` rather than the actual name of the optional dependency (`web-time`)...is this a standard practice for feature flag naming in the wasm ecosystem?

This was done in the past when [`stdweb`](https://crates.io/crates/stdweb) was still a thing, because there were two different Wasm Web backends: `wasm-bindgen` and `stdweb`.

But yes, the name refers to the Wasm Web backend.